mdpow/workflows/dihedrals.py

```diff
@@ -544,30 +614,65 @@ def plot_violins(df, resname, figdir=None, molname=None, width=0.9, solvents=SOL
     if molname is None:
         molname = resname
 
+        # figdir is now necessary??
```

Comment:
cleanup or address

Reply:
I will remove the comment and leave the convention as is for now. `figdir` will not be necessary, but will be automatically generated once the 'project manifest' and 'results directory structure' changes are made.